### ISSUE TITLE
Locking username by the first vote

### ISSUE THREAD

**Issue by fjacobi** [fgtclb/t3oodle#2]
Not a bug, but as discussed in Mattermost, the first vote will hard lock the username.
That makes it impossible for users to fix typos in their name or the usage of different nicknames, i.e. polls by different creators/interest groups.

**Comment by a-r-m-i-n** [comment]
True. This guest mode is only partially implemented. This locking only happens in JS. You could load an adopted t3oodle.js file, to change the behaviour.

Best is, to only allow logged in frontend users, to interact with polls, currently.
